frontend/packages/console-shared/src/components/status/StatusIconAndText.tsx

```diff
@@ -8,15 +8,22 @@ type StatusIconAndTextProps = {
   title?: string;
   spin?: boolean;
   iconOnly?: boolean;
+  noTooltip?: boolean;
 };
 
-const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({ icon, title, spin, iconOnly }) => {
+const StatusIconAndText: React.FC<StatusIconAndTextProps> = ({
+  icon,
+  title,
+  spin,
+  iconOnly,
+  noTooltip = false,
+}) => {
   if (!title) {
     return <>{DASH}</>;
   }
 
   return (
-    <span className="co-icon-and-text" title={title}>
+    <span className="co-icon-and-text" title={noTooltip ? '' : title}>
```

Comment:
As @christianvogt mentioned here https://github.com/openshift/console/pull/2561#issuecomment-528065553 the title should render only when iconOnly is used. Also using `undefined` instead of empty string results in title attribute not being present which is what we want.

```suggestion
    <span className="co-icon-and-text" title={iconOnly && !noTooltip ? title : undefined}>
```